demo/src/App.js

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+
+import {View} from "wonder-blocks-core";
+import Button from "wonder-blocks-button";
+import {Title} from "wonder-blocks-typography";
+import {ModalLauncher, OneColumnModal} from "wonder-blocks-modal";
+
+class App extends React.Component {
+    render() {
+        const modal = (
+            <OneColumnModal content={<Title>Hello, from modal</Title>} />
+        );
+
+        return (
+            <View style={styles.pinkSquare}>
+                <View style={styles.content}>
+                    <Title>Hello, world!</Title>
+                </View>
+                <View style={styles.row}>
+                    <ModalLauncher modal={modal}>
+                        {({openModal}) => (
+                            <Button onClick={openModal}>Open modal</Button>
+                        )}
+                    </ModalLauncher>
```

Comment:
Closing the modal for some reason breaks.  I'll look into it in a separate PR.  Here's the stack trace:
```
bundle.js:23601 Uncaught TypeError: this.updater.enqueueCallback is not a function
    at ModalLauncher.../node_modules/react/lib/ReactBaseClasses.js.ReactComponent.setState (bundle.js:23601)
    at Object.ModalLauncher._this._closeModal [as onCloseModal] (bundle.js:31428)
    at ModalBackdrop._this._handleClick (bundle.js:31142)
    at HTMLUnknownElement.boundFunc (bundle.js:15476)
    at Object.../node_modules/react-dom/lib/ReactErrorUtils.js.ReactErrorUtils.invokeGuardedCallback (bundle.js:15482)
    at executeDispatch (bundle.js:9155)
    at Object.executeDispatchesInOrder (bundle.js:9178)
    at executeDispatchesAndRelease (bundle.js:8568)
    at executeDispatchesAndReleaseTopLevel (bundle.js:8579)
    at Array.forEach (<anonymous>)
    at forEachAccumulated (bundle.js:21298)
    at Object.processEventQueue (bundle.js:8779)
    at runEventQueueInBatch (bundle.js:15515)
    at Object.handleTopLevel [as _handleTopLevel] (bundle.js:15525)
    at handleTopLevelImpl (bundle.js:15611)
    at ReactDefaultBatchingStrategyTransaction.perform (bundle.js:20514)
    at Object.batchedUpdates (bundle.js:15233)
    at Object.batchedUpdates (bundle.js:18482)
    at dispatchEvent (bundle.js:15686)
```